MANIFEST.in

```diff
@@ -1 +1,39 @@
 recursive-include unionml/templates *
+
+include README.md
+include unionml/py.typed  # marker file for PEP 561
+
+include CHANGELOG.md
+include changelog.md
+include LICENSE
+include LICENSE.txt
+include CITATION.cff
+include *.cff  # citation info
+
+include MANIFEST.in
+include pyproject.toml
+include setup.py
+include setup.cfg
+include tox.ini
+
+include requirements.txt
+recursive-include requirements *.txt
```

Comment:
what's this recursive-include for? the project currently has no `requirements` directory.

Reply:
I left it there for you to decide if you would like to move the dev and docs requirements into a `requirements` folder.

Reply:
> is the sed replacement in unionml/__init__.py still necessary after your changes?

No, it's not necessary. But did not want to drop it before discussing with you.